src/examples/signTaskEvaluatorPayout.js

```diff
@@ -0,0 +1,65 @@
+// An example using the setTaskEvaluatorPayout operation
+const signTaskEvaluatorPayout = async (colonyClient, taskId) => {
+
+  // Get JSON formatted operation from the mock database
+  const operationJSON = DATABASE.setTaskEvaluatorPayoutOperationJSON;
+
+  // Check the operation
+  if (operationJSON) {
+
+    // Restore operation
+    const operation = await colonyClient.setTaskEvaluatorPayout.restoreOperation(
+      operationJSON,
+    );
+
+    // Check if the operation matches the colony contract and the task id and
+    // check if current account is required to sign the operation.
+    if (
```

Comment:
As above, we should probably log something out if this doesn't work.

Reply:
:+1: